### DISCUSSION TITLE
Can't get OIT URP sample working with Windows build

### DISCUSSION THREAD

**Discussion by ghost** [happy-turtle/oit-unity#27] [Q&A]
Firstly, thanks for working on this great project, I really appreciate it!
I'm fairly new to Unity, so sorry if the things I ask are fairly obvious, but I followed the readme and can't get the OIT sample URP scene working.
These are the steps I follow:
1. Create a new URP project from the 3D (URP) blank template.
2. Add the OIT git package via the Package Manager.
3. Import the "URP Renderer Feature Demo" sample from the package and select it as the current scene.
4. Go to Settings/URP-HighFidelity-Renderer and add the Order Independent Rendering Renderer Feature.

The result is that the Unity Editor displays the sample scene correctly, but when I Build and Run the project for Windows Intel 64-bit, only the opaque objects are visible - the transparent ones are missing:
![image](https://github.com/happy-turtle/oit-unity/assets/20680486/c512ae7d-dcfe-4f17-aeb1-1bba0bfedb5a)

I've tried it first with Unity 2022.3.15f1 and figured that it must be a wrong version. Tried 2023.2.7f1 with the same result and then tried 2021.3.9f1 (which is mentioned in the project readme as the version that it was tested with) and couldn't even add the Renderer Feature due to script compilation errors:
```
Library\PackageCache\org.happy-turtle.order-independent-transparency@863657d0ad\URP\Runtime\OitPass.cs(31,98): error CS1061: 'ScriptableRenderer' does not contain a definition for 'cameraColorTargetHandle' and no accessible extension method 'cameraColorTargetHandle' accepting a first argument of type 'ScriptableRenderer' could be found (are you missing a using directive or an assembly reference?)
```

What am I doing wrong?

**Comment by happy-turtle** [comment]
Hi, thanks for trying out the project and reporting this. 😁 And a good reproduction one can follow. Sounds to me like the shader assets are missing in the build. I think you might actually be doing nothing wrong, this might be an issue of the current package setup. I will try to reproduce it.

**Comment by happy-turtle** [comment]
I was able to reproduce it, let's track in #28

**Comment by happy-turtle** [comment]
Fixed with https://github.com/happy-turtle/oit-unity/commit/91cdcf8610cfaf04e6b4c2c7e55d7abb054d4066
